.github/workflows/build-desktop-reusable.yml

```diff
@@ -43,7 +43,6 @@ jobs:
       win: ${{ steps.status.outputs.win }}
       mac: ${{ steps.status.outputs.mac }}
 
-    continue-on-error: true
```

Comment:
Does this mean that we don't want the report step to go ahead if the build doesn't complete?